components/locale-provider/index.tsx

```diff
@@ -20,7 +20,7 @@ export interface Locale {
 
 export interface LocaleProviderProps {
   locale: Locale;
-  children?: React.ReactElement<any>;
+  children?: React.ReactNode;
```

Comment:
`React.ReactChilren`

Reply:
`React.Children.only(this.props.children)` `only` require a parameter what type should be `ReactNode`.

Reply:
Hmm, you're right.